src/main/java/org/kuali/kra/negotiations/web/struts/action/NegotiationNegotiationAction.java

```diff
@@ -111,9 +109,15 @@ public ActionForward medusa(ActionMapping mapping, ActionForm form, HttpServletR
             // filtered with permissions. If that filter is removed, it will become an issue.
             super.loadDocument(negotiationForm);
         }
+
+        negotiationForm.getMedusaBean().setModuleName("neg");
+        negotiationForm.getMedusaBean().setMedusaViewRadio("0");
+        negotiationForm.getMedusaBean().setModuleIdentifier(negotiationForm.getNegotiationDocument().getNegotiation().getNegotiationId());
+        negotiationForm.getMedusaBean().generateParentNodes();
+
         //close the document overview as it can't be default closed via jsp.
-        negotiationForm.getTabStates().put("DocumentOverview", "false");
-        return mapping.findForward(Constants.MAPPING_BASIC);
+        //negotiationForm.getTabStates().put("DocumentOverview", "false");
```

Comment:
Why is this commented out? Do you need to close the document overview or no?

Reply:
Nope, the section is not in MedusaTab so probably don't need to close it. I'll take that line out.